### PR TITLE
Newsweek-Bootstrap

### PR DESCRIPTION
**Pull request by Diegodsha**
Project setup
Setting up a repository, HTML skeleton, images, icons, colors, and fonts.

Mobile-first approach
Working with Bootstrap on the mobile version and moving to tablet and desktop versions to make It correctly responsive.

Media queries
Use media queries to modify elements according to the version.

Deployment & documentation
Deploy Github pages, Create the read me file, check for linter errors, and create a pull request.

### CODE REVIEW

File: index.html
Comment (jubaan) [inline index.html]: - [x] Login, Login icon, and subscribe should use semantic HTML. They at as link or `<a>` tags on the original desing.

File: index.html
Comment (jubaan) [inline index.html]: - [x] Logo both int header and footer should be links with `<a>` anchor semantic HTML

File: index.html
Comment (jubaan) [inline index.html]: - [x] Search should use a form with a search input.

File: index.html
Comment (jubaan) [inline index.html]: - [x] These elements should be links

File: index.html
Comment (jubaan) [inline index.html]: - [x] Also these elements should be links

File: styles/desktop.css
Comment (jubaan) [inline styles/desktop.css]: - [x] Use bootstrap classes to minimize custom CSS stylesheets.
  - [x] This applies for text-colors, font-weights, list-style, borders, text-decoration, padding, margin, display flex.
  - [x] Review all stylesheets and look for these elements remove them from the custom CSS and adapt them with Bootstrap classes, the ones that apply.
  - [x] Review `desktop.css`, `mobile.ccs`, `style.css`, and `tablet.css` files.
  - [x] Also eliminate Duplicated code to keep the Stylesheets DRY and clean, to give an example of repeated code. `color: #fff` is repeated numerous times through the stylesheets. `padding-left:20`, `font-size: 18px`, and others. There are various ways to tackle this issue, by generating utility CSS classes, by grouping all elements that use the same rule and for some create global rules that apply to all elements.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Approved with OBSERVATIONS :recycle: 

## :wave: :wave: Hello @Diegodsha and @EroAuditore  . Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] The PR should have a descriptive message, a summary of the features, and things worth mentioning about the project.
- [ ] Feature branch should use a descriptive and meaningful name. Example `newsweek-clone`.
- [x] Commit messages should be meaningful and descriptive. Example `Add navbar feature`, `Merge navbar-prototype with newsweek-clone branch`, `Update contact info in README`.
- [x] The live preview link in README is broken.
- [x] The MIT license link takes to a Wikipedia page that explains the MIT license. Please create your MIT LIcense file and link it in the REAME.
- [x] Change the social media handlers to the actual username. Example: #BAD [@githubhandle](#) #GOOD [@jubaan](https://www.github.com/jubaan)
- [x] The preview screenshot is not displaying in the README
- [x] Wrap the `<header>` with semantic HTML
- [x] See inline comments for `index.html` and stylesheet files.
- [ ] Navbar
  - [ ] DESKTOP view
    - [x] the date the logo and login and subscribe elements are not aligned as the original design. They should be vertically aligned to the end or bottom in line with the bottom of the letter in the newsweek logo.
    - [x] The subscribe link should have a space to the right so it doesn't touch the border of the screen.
    - [x] The search and its icon should be part of the same element.
    - [x] The navbar has a margin-top that creates a white space between the navbar and the top of the browser.  
    - [x] the navbar logo has a different background color that dont' match the design. Please try to find in the original website the svg image.
    image
  - [ ] MOBILE view
    - [x] elements are not equally spaced to the margins
    image

- [ ] Footer
  - [ ] DESKTOP
    - [x] logo, copyright and social media icons are not aligned correctly, vertically and horizontally.
    - [x] Editions section is taking another row to display the information, all should be in the same line
    - [x] Logo, editions section and about us section should all be aligned equally to the left
  image
  - [ ] TABLET and MOBILE views
    - [x] The same applies for the tablet and mobile views
  image
  image

**NOTE**: the milestone should appear as approved in your Microverse progress dashboard, this doesn't mean that you shouldn't work on fixing the previous observations, so please work on them before moving to the next milestone. If you don't do it, observations will pile up on Milestone 3.





## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [x] You're supposed to use Bootstrap CSS classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)). The main goal of this project is to learn how to **use the Bootstrap framework**, specifically its responsive design features: you should focus on this aspect as you work on your project. That being said, use bootstrap to replace most of your media queries and other classes in your `style.css`. 

## Please Note: ⚠️ 

1. This request will be marked as approved in your dashboard
2.  You don’t need to submit it for re-review or merge this request into the main branch 
3. Make the suggested changes and move on to the next milestone.  
4. Submit this same pull request again, don't create a new pull request when you're done with the next milestone, you are to keep submitting this same pull request until project completion.

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

File: styles/desktop.css
Comment (euqueme) [inline styles/desktop.css]: - [x] Use bootstrap classes for basic colors (like white, black or red)

File: styles/desktop.css
Comment (euqueme) [inline styles/desktop.css]: - [x] Theres a bootstrap class for font-weight let's use it

File: styles/desktop.css
Comment (euqueme) [inline styles/desktop.css]: - [x] Use the bootstrap class for `list-style: none`

File: styles/desktop.css
Comment (euqueme) [inline styles/desktop.css]: - [x] Use the bootstrap class for `text-decoration: none`

File: styles/desktop.css
Comment (euqueme) [inline styles/desktop.css]: - [x] Use bootstrap classes for basic colors (like white, black or red)

File: styles/desktop.css
Comment (euqueme) [inline styles/desktop.css]: - [x] Use bootstrap classes for basic colors (like white, black or red)

File: styles/desktop.css
Comment (euqueme) [inline styles/desktop.css]: - [x] Use the bootstrap class for `text-decoration: none`

File: styles/style.css
Comment (euqueme) [inline styles/style.css:11]: - [x] Use the bootstrap class for `text-decoration: none`

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use the bootstrap class for `text-decoration: none`

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use the bootstrap class for `text-decoration: none`

File: styles/mobile.css
Comment (euqueme) [inline styles/mobile.css]: - [x] Use bootstrap classes for basic colors (like white, black or red)

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use bootstrap classes for basic colors (like white, black or red)

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use bootstrap classes for basic colors (like white, black or red)

File: styles/tablet.css
Comment (euqueme) [inline styles/tablet.css]: - [x] Use bootstrap classes for basic colors (like white, black or red)

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use bootstrap classes for `display`

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use bootstrap classes for `display`

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use bootstrap classes for `display`

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use the bootstrap class for `position: sticky`

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use the bootstrap class for text-transform

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use the bootstrap class for text-align

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use the bootstrap class for `position`

File: styles/style.css
Comment (euqueme) [inline styles/style.css]: - [x] Use the bootstrap class for `position`

File: styles/style.css
Comment (euqueme) [inline styles/style.css:49]: - [x] Use the bootstrap class for `position`

File: styles/style.css
Comment (euqueme) [inline styles/style.css:58]: - [x] Use the bootstrap class for `position`

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)
### Hello @Diegodsha and @EroAuditore :lollipop: :balloon:  

### Thank you for your submission :handshake: your project looks amazing :astonished: is almost like the original :star_struck: claps for the great work done so far :clap: :clap: :clap: let's make the following changes :construction_worker: 

### General

- [x] Use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)
> make use of the aside tag for the sections below the editor's picks.

### Implementation requirements

* [x] Use Bootstrap CSS classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))

> I've put comments directly in the CSS file, where you should use bootstrap classes and the CSS files should be as small as possible

### All Milestone requirements

#### Mobile Screens

##### TOP STORY
- [x] This should be the first section showing in this screen size this is usually achievable with the `flex-column-reverse` bootstrap class
Clone
image
Original
image
##### Main news sections (U.S., World, etc.)
- [x] The images of the small articles should take 60% of the screen width
- Clone
image
- Original
image
##### SUBSCRIBE
- [x] The images, text and subscribe button should be centered and the bold text should be on top of the images
- Clone
image
- Original
image
##### NEWSLEETTER
- [x] The text should be centered, and the container that holds the text, email input field and FREE SIGN UP button should have a gray border
- Clone
image
- Original
image
##### Footer
- [x] The  last 3 rows with links should also have a white background and should be aligned  with the Newsweek logo and social media icons
- Clone
image
- Original
image
#### Tablet Screens
##### NEWSLETTER
- [x] The text should be centered
Clone
image
Original
image
##### Footer
- [x] The footer should have less margin on the left and right in comparison with the upper sections
Clone
image
Original
image
#### Desktop Screens
##### SUBSCRIBE and NEWSLETTER
- [x] should have more margin on the left and right in comparison with the Footer
Clone
image
Original
image
#### All screen sizes
##### Column 2 and 3
* [x] Each image have a link
* [x] Each title uses a hx tag and have a link
> All the images starting from MY TURN on column 2 to OPINIONS on column 3 are missing these let's fix them.

### That's it, team!  :raised_hands: 
### Please :pray:  do the changes :technologist: :technologist:  and submit back for a code review when you are done :memo: 
Any question? ask me on slack **@ Maru** :woman_technologist: 
### Happy coding! :rocket: 
PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 

- Your project is almost perfect. Excellent work applying Maru's requested changes.

## However, there are some changes required 👀 🧰 🔨 🔧 
- On Desktop view, the images of your in-magazine section are distorted. Please fix this:
image

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Pull request merged**
